simtools/util/names.py

```diff
@@ -42,19 +54,21 @@ def validate_camera_name(name):
     """
     Validate a camera name.
 
-    Raises
-    ------
-    ValueError
-        If name is not valid.
-
     Parameters
     ----------
-    name: str
+    name: (str, required)
+        Camera name
 
     Returns
     -------
     str
         Validated name.
+
+    Raises
```

Comment:
Technically, this is not correct. This method does not raise any exceptions (it is in `validate_name`). Comments applies to all similar methods.

Reply:
Ok, I thought we could propagate it. I guess this would be the case, when the Raise statement is inside a private function, which does not make it to the API, right? Anyway, it is not the case here. I removed the misplaced Raises.

Reply:
I don't see the changes, but let's leave it. Propagating means would mean what? One level, two levels, up to the applications - I guess you see the issue. Best to document where it happens, and that's it.